features/dp-cantabular-csv-exporter-user-published.feature

```diff
@@ -6,96 +6,96 @@ Feature: Cantabular-Csv-Exporter-Published
     Given the following response is available from Cantabular from the codebook "Example" using the GraphQL endpoint:
       """
       {
-	"data": {
-	    "dataset": {
-		"table": {
-		    "dimensions": [
-			{
-			    "categories": [
-				{
-				    "code": "0",
-				    "label": "London"
-				},
-				{
-				    "code": "1",
-				    "label": "Liverpool"
-				},
-				{
-				    "code": "2",
-				    "label": "Belfast"
-				}
-			    ],
-			    "count": 3,
-			    "variable": {
-				"label": "City",
-				"name": "city"
-			    }
-			},
-			{
-			    "categories": [
-				{
-				    "code": "0",
-				    "label": "No siblings"
-				},
-				{
-				    "code": "1",
-				    "label": "1 sibling"
-				},
-				{
-				    "code": "2",
-				    "label": "2 siblings"
-				},
-				{
-				    "code": "3",
-				    "label": "3 siblings"
-				},
-				{
-				    "code": "4",
-				    "label": "4 siblings"
-				},
-				{
-				    "code": "5",
-				    "label": "5 siblings"
-				},
-				{
-				    "code": "6",
-				    "label": "6 or more siblings"
-				}
-			    ],
-			    "count": 7,
-			    "variable": {
-				"label": "Number of siblings",
-				"name": "siblings"
-			    }
-			}
-		    ],
-		    "error": null,
-		    "values": [
-			1,
-			0,
-			0,
-			1,
-			0,
-			0,
-			0,
-			0,
-			0,
-			0,
-			0,
-			1,
-			0,
-			0,
-			0,
-			0,
-			1,
-			0,
-			0,
-			1,
-			1
-		    ]
-		}
-	    }
-	}
+        "data": {
```

Comment:
what has changed here?

Reply:
It is just indenting I think, my emacs always removes unnecessary tab or space characters